vignettes/laminr.Rmd

```diff
@@ -25,7 +25,7 @@ This vignette introduces the basic **{laminr}** workflow.
 Install **{laminr}** from CRAN:
 
 ```r
-install.packages("laminr")
+install.packages("laminr", dependencies = TRUE)
```

Comment:
I usually wouldn't suggest doing this here but it's fine for now

Reply:
We have it in the quickstart and in the demo instruction.

Reply:
If we don't have it here, it'd be inconsistent with the other training material.

The goal is to have a one line installation command and not talk through different ways of installing the package.